### PR TITLE
Test amazon books

### PR DESCRIPTION
**Pull request by ilonamaslova**

### CODE REVIEW

File: src/test/java/testsAmazon/BooksTests/BooksTests.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    @Test
+    public void checkAllElementsBooks() {
+        HomePage homePage = new HomePage(driver);
+        homePage.navigate();
+        homePage.getBtnAllInHeader().click();
+        homePage.getBtnBooks().click();
+        homePage.getBtnSearch().click();
+        BooksPage booksPage = new BooksPage(driver);
+
+        sleep(3);
```
Comment (StepaniukAndii) [inline src/test/java/testsAmazon/BooksTests/BooksTests.java:22]: Why use a sleep() instead explicit wait?

File: src/test/java/testsAmazon/BooksTests/BooksTests.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        Assert.assertTrue(booksInSpanishPage.getHeaderText().getText().contains("Libros en Español"));
+        sleep(3);
+        Assert.assertTrue(driver.getTitle().contains("Libros"));
+
+        booksInSpanishPage.getBasketBtn().click();
+        BasketAmazonPage basketAmazonPage = new BasketAmazonPage(driver);
+
+        Assert.assertTrue(basketAmazonPage.getImg().isDisplayed());
+        Assert.assertTrue(basketAmazonPage.getBtnSell().getText().contains("Sell"));
```
Comment (StepaniukAndii) [inline src/test/java/testsAmazon/BooksTests/BooksTests.java:37]: use Assert.assertEquals()

File: src/test/java/testsAmazon/BooksTests/BooksTests.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        waitTILLELelementContainsText("//a[@class='nav_a nav-swm-text nav-progressive-attribute nav-progressive-content']", "Shop Mother's Day Gifts");
+        Assert.assertTrue(basketAmazonPage.getBtnShopMothersDayGifts().getText().contains("Shop Mother's Day Gifts"));
```
Comment (StepaniukAndii) [inline src/test/java/testsAmazon/BooksTests/BooksTests.java:39]: use Assert.assertEquals()

File: src/test/java/testsAmazon/BooksTests/BooksTests.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        sleep(3);
+        Assert.assertTrue(booksPage.getHeaderText().getText().contains("Popular in Books"));
+
+        booksPage.getLinkBooksInSpanish().click();
+        BooksInSpanishPage booksInSpanishPage = new BooksInSpanishPage(driver);
+
+        sleep(3);
+        Assert.assertTrue(booksInSpanishPage.getHeaderText().getText().contains("Libros en Español"));
+        sleep(3);
+        Assert.assertTrue(driver.getTitle().contains("Libros"));
```
Comment (StepaniukAndii) [inline src/test/java/testsAmazon/BooksTests/BooksTests.java:31]: use Assert.assertEquals()

File: src/test/java/testsAmazon/BooksTests/BooksTests.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        BooksPage booksPage = new BooksPage(driver);
+
+        sleep(3);
+        Assert.assertTrue(booksPage.getHeaderText().getText().contains("Popular in Books"));
+
+        booksPage.getLinkBooksInSpanish().click();
+        BooksInSpanishPage booksInSpanishPage = new BooksInSpanishPage(driver);
+
+        sleep(3);
+        Assert.assertTrue(booksInSpanishPage.getHeaderText().getText().contains("Libros en Español"));
```
Comment (StepaniukAndii) [inline src/test/java/testsAmazon/BooksTests/BooksTests.java:29]: use Assert.assertEquals()

File: src/test/java/testsAmazon/BooksTests/BooksTests.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+public class BooksTests extends TestInit {
+
+    @Test
+    public void checkAllElementsBooks() {
+        HomePage homePage = new HomePage(driver);
+        homePage.navigate();
+        homePage.getBtnAllInHeader().click();
+        homePage.getBtnBooks().click();
+        homePage.getBtnSearch().click();
+        BooksPage booksPage = new BooksPage(driver);
```
Comment (StepaniukAndii) [inline src/test/java/testsAmazon/BooksTests/BooksTests.java:20]: better when all created objects are in one place

File: src/test/java/testsAmazon/BooksTests/BooksTests.java [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+        homePage.navigate();
+        homePage.getBtnAllInHeader().click();
+        homePage.getBtnBooks().click();
+        homePage.getBtnSearch().click();
+
+        Assert.assertEquals("Popular in Books", booksPage.getHeaderText().getText());
+
+        booksPage.getLinkBooksInSpanish().click();
+        sleep(3);
+//        assertEquals("text", driver.findElement(By.cssSelector(".title")).getText());
```
Comment (StepaniukAndii) [inline src/test/java/testsAmazon/BooksTests/BooksTests.java:29]: delete commit code

File: src/test/java/testsAmazon/BooksTests/BooksTests.java [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+        Assert.assertEquals("Libros en Español", booksInSpanishPage.getHeaderText().getText());
+        sleep(3);
```
Comment (StepaniukAndii) [inline src/test/java/testsAmazon/BooksTests/BooksTests.java:31]: delete sleep() use exsplicyt wait

File: hs_err_pid7912.log [hunk 9]
```diff
@@ -0,0 +1,526 @@
...
+CPU: total 4 (initial active 4) (4 cores per cpu, 1 threads per core) family 18 model 1 stepping 0 microcode 0x0, cx8, cmov, fxsr, mmx, 3dnowpref, sse, sse2, sse3, sse4a, popcnt, lzcnt, tsc, tscinvbit, clflush
+
+Memory: 4k page, system-wide physical 3563M (328M free)
+TotalPageFile size 9902M (AvailPageFile size 8M)
+current process WorkingSet (physical memory assigned to process): 57M, peak: 57M
+current process commit charge ("private bytes"): 143M, peak: 143M
+
+vm_info: OpenJDK 64-Bit Server VM (17.0.2+8-86) for windows-amd64 JRE (17.0.2+8-86), built on Dec  7 2021 21:49:10 by "mach5one" with MS VC++ 16.8 / 16.9 (VS2019)
+
+END.
```
Comment (StepaniukAndii) [inline hs_err_pid7912.log:526]: What is it?

**Review by StepaniukAndii — APPROVED** [review]

**Review by StepaniukAndii — APPROVED** [review]

**Review by Bogdan27nazaruk — APPROVED** [review]

**Pull request merged**